src/sentry/tasks/reprocessing2.py

```diff
@@ -119,31 +133,52 @@ def reprocess_event(project_id, event_id, start_time):
     time_limit=(60 * 5) + 5,
     soft_time_limit=60 * 5,
 )
-def wait_group_reprocessed(project_id, group_id):
+def wait_group_reprocessed(project_id, group_id, new_group_id):
     from sentry.reprocessing2 import is_group_finished
 
     if is_group_finished(group_id):
-        delete_old_group.delay(project_id=project_id, group_id=group_id)
+        finish_reprocessing.delay(
+            project_id=project_id, group_id=group_id, new_group_id=new_group_id
+        )
     else:
         wait_group_reprocessed.apply_async(
-            kwargs={"project_id": project_id, "group_id": group_id}, countdown=60 * 5
+            kwargs={"project_id": project_id, "group_id": group_id, "new_group_id": new_group_id},
+            countdown=60 * 5,
         )
 
 
 @instrumented_task(
-    name="sentry.tasks.reprocessing2.delete_old_group",
+    name="sentry.tasks.reprocessing2.finish_reprocessing",
     queue="events.reprocessing.preprocess_event",
     time_limit=(60 * 5) + 5,
     soft_time_limit=60 * 5,
 )
-def delete_old_group(project_id, group_id):
-    from sentry import similarity
-    from sentry.models.group import Group
+def finish_reprocessing(project_id, group_id, new_group_id):
```

Comment:
You should technically keep the old task around, but given that there is basically no usage on this at the moment it should be safe to remove the old task right away.

Reply:
i'm definetly yoloing that part especially with the changed function signatures across the board